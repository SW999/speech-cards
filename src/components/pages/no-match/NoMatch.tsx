import * as React from 'react';
import { FunctionComponent } from 'react';

export const NoMatch: FunctionComponent = () => {
  return (
    <div className="no-match-wrapper">
      <h1>404</h1>
      <h2>Ups, not found!</h2>
    </div>
  );
};
