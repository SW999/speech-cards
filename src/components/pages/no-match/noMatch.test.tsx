import * as React from 'react';
import { render } from '@testing-library/react';
import { NoMatch } from './NoMatch';

describe('NoMatch component', () => {
  it('NoMatch renders correctly', () => {
    const { asFragment } = render(<NoMatch />);

    expect(asFragment()).toMatchSnapshot('NoMatch');
  });
});
