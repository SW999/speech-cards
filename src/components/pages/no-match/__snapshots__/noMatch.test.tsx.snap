// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch component NoMatch renders correctly: NoMatch 1`] = `
<DocumentFragment>
  <div
    class="no-match-wrapper"
  >
    <h1>
      404
    </h1>
    <h2>
      Ups, not found!
    </h2>
  </div>
</DocumentFragment>
`;
